python/cog/schema.py

```diff
@@ -57,6 +64,7 @@ class PredictionBaseModel(pydantic.BaseModel, extra=pydantic.Extra.allow):
 
 
 class PredictionRequest(PredictionBaseModel):
+    input: Dict[str, Any]
```

Comment:
In the cog original implementation, the `input` is in the `PredictionBaseModel`, but we don't want it existed in our `NewPredictionBaseModel`, so we move it here.
If without this field, some tests will fail.